drafts/developer-guide/pages/corteza-duplicate-detection/index.adoc

```diff
@@ -0,0 +1,61 @@
+include::ROOT:partial$variables.adoc[]
+
+= Corteza duplicate detection
+
+:keywords: API, {PRODUCT_NAME} API, {PRODUCT_NAME}
+:description: Learn how to USE {PRODUCT_NAME} duplicate detection to maintain clean and accurate data.
+
+{PRODUCT_NAME} through the Duplicate detection feature helps maintain clean and accurate data. {PRODUCT_NAME} gives you tools for managing duplicates one at a time and across your organization, and to track your progress in eliminating duplicates.
+
+image::duplication-detection/duplicate-detection.gif[role="data-zoomable"]
+
+[NOTE]
+The duplicate detection feature can be used in  creating or editing a record, on existing records, importing records, etc. 
+
+There are two main distinctive Categories:
+    
+    * on creating and updating a record/records (active duplicate detection)
+    * on checking for duplicates on existing records
```

Comment:
This is not true; you currently can't check an existing record list for duplicates.
Double check with @vicpatel